### ISSUE TITLE
Would you still like help with automation?

### ISSUE THREAD

**Issue by rfaile313** [blavejr/OrganiseDesktop#24]
If so, how do you want to automate it? Do you want to run a timer in the script and leave it running, or create a bash script that runs the script every day?

**Comment by blavejr** [comment]
hi @rfaile313, I think a bash script would be a better solution, take a crack at it please.

**Comment by rfaile313** [comment]
Will do, I'm on it!

**Comment by blavejr** [comment]
Brilliant.

**Comment by rfaile313** [comment]
Alright,

I've added a simple bash script to run your desktop cleaner. 
[RunDesktopCleanerDaily.zip](https://github.com/blavejr/OrganiseDesktop/files/1530410/RunDesktopCleanerDaily.zip)
The first path specifies the location of the Python version you're looking to use (helpful, because many Unix systems have python2 installed and your script is Python 3+. The second is the location of the Python script itself.

I think the best bet is to implement a cron job to run the bash script daily. It's really easy, and will run over any UNIX based system (Linux, Mac, etc.)

cron will run the shell script which in turn runs the desktop cleaner script periodically at fixed times, dates, or intervals.

In the command line run:

`crontab -e`

This will open the vi editor, and you will need to add this:

`0 18 * * * /home/MyUser/MyCrons/RunDesktopCleanerDaily.sh`

This, for example, this will execute the Desktop Cleaner Python script every day at 6 PM.

I think this solves your automation issue, but let me know what you think and/or if you have any questions!

**Comment by blavejr** [comment]
@rfaile313 please send a PR request so I can review.

**Comment by rfaile313** [comment]
There's no compare, these are all external features that you will have to describe and setup for people to be able to automate your script.

**Comment by blavejr** [comment]
Oh alright, its great than. as long as we have made some progress on it. we can work on making it simpler later.

**Comment by rfaile313** [comment]
No doubt. I'd like to continue helping, let's keep chatting. I'm not sure it gets much easier than this though.

**Comment by Sudhanshu36** [comment]
Great progress.

**Comment by blavejr** [comment]
Haha @rfaile313 just read the code and add anything you feel is needed,  whatever you want to add, you are free!

**Comment by rfaile313** [comment]
What's the status on this? Did you still want to try and do it without a bash script/cronjob?

**Comment by blavejr** [comment]
yeah I want to try it but you haven't sent in a pr request. How do I get your files?

**Comment by omerharshit** [comment]
I want to contribute as an python devloper.

**Comment by dhamodaran-pandiyan** [comment]
Is this issue still active? If so, please share the progress, otherwise people can start work on it.
